### PR TITLE
doc: nRF54H SUIT img update

### PR DESCRIPTION
**Pull request by FrancescoSer**
Updated 54H SUIT doc images.

### CODE REVIEW

**Comment by maciejpietras** [comment]
For consistency -  Should we use SDRFW instead of SDFW_Recovery?

**Review by annwoj — APPROVED** [review]

**Review by richabp — COMMENTED** [review]
In Fig "nrf54h/images/nrf54h20_recovery_invocation_topology.png", shouldn't one box under Radio domain, be **Radio Recovery** as mentioned in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_device_config.html#id13.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/images/nrf54h20_invocation_topology.svg
Comment (wiba-nordic) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/images/nrf54h20_invocation_topology.svg:1]: Text alignment in the legend box seems off.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/images/nrf54h20_recovery_invocation_topology.svg
Comment (melwee01) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/images/nrf54h20_recovery_invocation_topology.svg:1]: @FrancescoSer the corrected version of this image is in your email

**Review by richabp — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**